applications/StructuralMechanicsApplication/tests/adjoint_sensitivity_analysis_tests/adjoint_beam_structure_3d2n/beam_test_nodal_disp_adjoint_parameters.json

```diff
@@ -121,7 +121,7 @@
             "tolerance"        : 1e-5,
             "time_frequency"   : -2.0
         }
-    }],
+    }]},
```

Comment:
This looks strange, does everything work if you just remove the "_" from "_json_output_process"?

Reply:
the process has to be out of the scope of `processes` otherwise it will be constructed regardless of the name